### PR TITLE
Add changelog for 0.16 releases.

### PR DESCRIPTION
**Pull request by lukehoban**
See also #612 for corresponding API doc updates.

### CODE REVIEW

File: reference/changelog.md [hunk 1]
```diff
@@ -71,6 +76,95 @@ redirect_from: /install/changelog.html
...
+
+### Pulumi CLI
+
+Major features of this release include:
+
+#### Improvements to CLI output
+
+Default colors that fit better for both light and dark terminals.  Overall updates to rendering of previews/updates for consistentcy and simplicity of the display. 
+
+#### Parallelized ressource deletion
```
Comment (justinvp) [inline reference/changelog.md]: ressource => resource

File: reference/changelog.md [hunk 2]
```diff
@@ -71,6 +76,95 @@ redirect_from: /install/changelog.html
...
+- Pass logging messages to the right sink when doing docker operations. [pulumi/pulumi-docker#7](https://github.com/pulumi/pulumi-docker/pull/7)
+- Add `docker.Image` component. [pulumi/pulumi-docker#10](https://github.com/pulumi/pulumi-docker/pull/10)
+
+### @pulumi/gcp 0.16.0
+
+- Adopted `v1.15.0` of the Google Terraform Provider. [pulumi/pulumi-gcp#51](https://github.com/pulumi/pulumi-gcp/pull/51)
+
+### @pulumi/kubernetes 0.17.0
+
+TODO: @hausdroff
```
Comment (justinvp) [inline reference/changelog.md]: TODO @hausdorff

File: dashboard.json [hunk 3]
```diff
@@ -36,6 +36,11 @@
         "title": "News",
         "icon": "radio",
         "items": [
+            {
+                "title": "Pulumi 0.16.0 now available",
```
Comment (justinvp) [inline dashboard.json]: Updating this file is no longer necessary (though, there's no harm in doing so). I'm planning to delete this file in the near future, along with the instructions on updating it.

File: reference/changelog.md [hunk 4]
```diff
@@ -71,6 +76,95 @@ redirect_from: /install/changelog.html
...
+
+### @pulumi/vsphere 0.15.0
```
Comment (ellismg) [inline reference/changelog.md]: This section is here twice, is that by design?

File: reference/changelog.md [hunk 5]
```diff
@@ -71,6 +76,95 @@ redirect_from: /install/changelog.html
...
+
+Added a new provider for VMware vSphere, enabling deployment and management of resources in vCenter Server and ESXi. Thanks [@Smithx10](https://github.com/Smithx10)!
+
+### @pulumi/vsphere 0.15.0
```
Comment (ellismg) [inline reference/changelog.md]: Should we rebuild OpenStack, vSphere and eks with the 0.16.0 Pulumi references?

Comment (lukehoban) [reply reference/changelog.md]: Yes - let's go ahead and do this for consistency.

File: reference/changelog.md [hunk 6]
```diff
@@ -71,6 +76,91 @@ redirect_from: /install/changelog.html
...
+
+In addition to the 0.16.0 CLI release, we've released new versions of all of our packages.
+
+### Pulumi CLI
+
+Major features of this release include:
+
+#### Improvements to CLI output
+
+Default colors that fit better for both light and dark terminals.  Overall updates to rendering of previews/updates for consistentcy and simplicity of the display.
```
Comment (wolverian) [inline reference/changelog.md]: Typo:

```suggestion
Default colors that fit better for both light and dark terminals.  Overall updates to rendering of previews/updates for consistency and simplicity of the display. 
```

File: reference/changelog.md [hunk 7]
```diff
@@ -71,6 +76,91 @@ redirect_from: /install/changelog.html
...
+- Updated to latest `@pulumi/pulumi` and `@pulumi/aws` dependencies.
+
+### @pulumi/aws-serverless 0.16.0
+
+- This library is now deprecated.  All features of this library are now avaiable directly in `@pulumi/aws` (see above).  In a future update the same change will be made for `@pulumi/azure-serverless` moving similar functionality into `@pulumi/azure`.
+
+### @pulumi/azure 0.16.0
+
+- Adopted `v1.15.0` of the AzureRM Terraform Provider. [pulumi/pulumi-azure#125](https://github.com/pulumi/pulumi-azure/pull/125)
+- Many iprovements to autoname generation to match Azure resource name requirements.
```
Comment (wolverian) [inline reference/changelog.md]: Typo:

```suggestion
- Many improvements to autoname generation to match Azure resource name requirements.
```

**Pull request merged**